service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
-In case of issues you can remove your KafkaConnection by executing
-`oc delete akc name-of-your-kafka` and retry operation
-endif::[]
+. Get the URL of the Route created for the application.
++
+[source,options="nowrap"]
+----
+$ oc get route
+
+NAME                   HOST/PORT
+rhoas-quarkus-kafka    rhoas-quarkus-kafka-jbyrne-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com
+----
+
+. In your terminal, highlight the URL shown under *HOST/PORT*. Right-click in the terminal window and  select *Copy*.
 
-[id="proc-create-prices-topic-for-operator_{context}"]
-== Creating the prices Kafka topic in {product}
+. In your web browser, paste the URL for the Route.
++
+A web page for the Quarkus application opens.
 
-For this quick start, the Kafka topic that the Quarkus example application references is called `prices`. You need to create this topic in {product} so that the Quarkus application can interact with it.
+. In your web browser, append `prices.html` to the URL.
++
+A new web page, entitled *Last price*, opens.  Because you have not yet connected the Quarkus application to your Kafka instance, the price value appears as `N/A`.
+
+== Step 4: Creating a topic in your Kafka instance
+
+[role="_abstract"]
+In the previous step of this tutorial, you created an example OpenShift application. The application is a Quarkus application that requires a Kafka topic called `prices`. In this step, you create the `prices` topic in your Kafka instance so that the Quarkus application can interact with it.
 
 .Prerequisites
-* You've created a Kafka instance in {product} and the instance is in *Ready* state.
+* You have completed the previous steps in this tutorial:
+** *Step 1: Verifying connection to your OpenShift cluster*
+** *Step 2: Connecting a Kafka instance to your OpenShift cluster*
+** *Step 3: Deploying an example application in OpenShift*
+ifndef::community[]
+* You have a Kafka instance in {product-long} that is in the *Ready* state. To learn how to create a Kafka instance, see link:https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/f351c4bd-9840-42ef-bcf2-b0c9be4ee30a[Getting started with Streams for Apache Kafka].
+endif::[]
+ifdef::community[]
+* You have a Kafka instance in {product-long} that is in the *Ready* state. To learn how to create a Kafka instance, see link:https://github.com/redhat-developer/app-services-guides/tree/main/getting-started[Getting started with {product-long}].
+endif::[]
 
 .Procedure
-. In the *Streams for Apache Kafka* page of the web console, select the name of the Kafka instance that you want to add a topic to.
+. On the *Kafka instances* page of the web console, click the name of the Kafka instance that you want to add a topic to.
+
 . Click *Create topic* and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
 +
-[.screencapture]
 .Guided steps to define topic details
```

Comment:
I will ping @sterobin as I think we have this common step across different guides that were refined recently. 
Worth to take look if we have this unified.

Reply:
@wtrocki I will discuss this further with Stetson/CCS. However, this content _does_ deviate a little from the content included in the quick starts.